tests/integration/update_cluster/restrict_access/id_rsa.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAAAgQCtWu40XQo8dczLsCq0OWV+hxm9uV3WxeH9Kgh4sMzQxNtoU1pvW0XdjpkBesRKGoolfWeCLXWxpyQb1IaiMkKoz7MdhQ/6UKjMjP66aFWWp3pwD0uj0HuJ7tq4gKHKRYGTaZIRWpzUiANBrjugVgA+Sd7E/mYwc/DMXkIyRZbvhQ==
```

Comment:
Who owns this key? What is it for?

Reply:
This is the same key that is used by all integration tests. For example, the [shared vpc test]. If you remove it, the test will fail with the following error:
```
--- FAIL: TestRestrictAccess (0.01s)
    integration_test.go:238: error running "in-v1alpha2.yaml" create: error reading SSH public key
    ....
```


[shared vpc test]: https://github.com/kubernetes/kops/blob/master/tests/integration/update_cluster/shared_vpc/id_rsa.pub

Reply:
Yeah, it's a little annoying, but it's just a one-off key I generated.  It isn't used for anything but the tests.